raiden_installer/installer_parts/raiden.py

```diff
@@ -0,0 +1,106 @@
+import os
```

Comment:
You can rewrite these to either use `contextlib.contextmanager` or the magic methods `__enter__` and `__exit__` of a class , or even a class which will make the code alot cleaner. Take a look at the original code, there's an example of that there.